develop/dev-guide-tidb-crud-sql.md

```diff
@@ -19,7 +19,23 @@ Please make sure you are connected to a TiDB cluster. If not, refer to [Build a
 
 TiDB is compatible with MySQL, you can use MySQL statements directly in most cases. For unsupported features, see [Compatibility with MySQL](/mysql-compatibility.md#unsupported-features).
 
-To experiment with SQL and test out TiDB compatibility with MySQL queries, you can [run TiDB directly in your web browser without installing it](https://tour.tidb.io/). You can also first deploy a TiDB cluster and then run SQL statements in it.
+To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try the SQL editor in your [TiDB Cloud console](https://tidbcloud.com/). Navigate to the **Cluster Details** page and click **SQL Editor** on the left navigation bar. In the SQL editor, you can edit and run SQL queries directly against your cluster without using a terminal.
```

Comment:
```suggestion
To experiment with SQL and test out TiDB compatibility with MySQL queries, you can try the SQL editor in your [TiDB Cloud console](https://tidbcloud.com/). Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and click **SQL Editor** on the left navigation bar. In the SQL editor, you can edit and run SQL queries directly against your cluster without using a terminal.
```

Reply:
SQL editor is only available inside the individual cluster page.